SignalMessaging/categories/UIDevice+featureSupport.swift

```diff
@@ -11,7 +11,9 @@ public extension UIDevice {
     }
 
     @objc
-    public var isIPhoneX: Bool {
+    public var hasIPhoneXNotch: Bool {
+        return false
```

Comment:
Don't we only want to return false if we're on iOS 12 or later?  

If not, can we remove the logic below?

Reply:
Oh I see - we only call this after an `@available` check.  I wonder if we should enforce that with a `guard` here?

Reply:
Hmm, it still seems like you may have left in this `false` by accident?

Reply:
🤦‍♂️